### PR TITLE
nano bundle

### PR DESCRIPTION
**Pull request by TechnologyClassroom**
I have tested this with a full build on Ubuntu 16.04.

nano bundle requires libncursesw so I had to modify ncurses as well.  All functionality with util-linux is maintained through symlinks.

### CODE REVIEW

**Comment by ivandavidov** [comment]
I will test this more thoroughly in December.

@TechnologyClassroom - can you provide more descriptive information in ``.config`` and the readme file what other dependencies (e.g. ncurses, probably other as well) are required when we build nano?

The rising issue is that the more software we have, more dependencies there are to handle. For now the simplest solution is to mention somewhere the explicit dependencies we need (i.e. in the '.config' file and in the readme document).

**Comment by TechnologyClassroom** [comment]
I can work on that.  I was mostly just pasting the description from the original projects.

nano only requires build-essentials and ncurses I believe.  I will do a quick .config and readme update before the next bundle.

File: src/minimal_overlay/bundles/nano/02_build.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+echo "Preparing nano work area. This may take a while..."
+make -j $NUM_JOBS clean
+
+rm -rf $DESTDIR
+
+echo "Configuring nano..."
+CFLAGS="$CFLAGS" ./configure \
+    --prefix=/usr
+    --disable-utf8 \
+    CFLAGS="-Os -s -fno-stack-protector -U_FORTIFY_SOURCE"
```
Comment (bauen1) [inline src/minimal_overlay/bundles/nano/02_build.sh]: Harcoded CFLAGS, please replace this with `$CFLAGS`

File: src/minimal_overlay/bundles/ncurses/02_build.sh [hunk 2]
```diff
@@ -25,13 +25,17 @@ echo "Configuring Ncurses..."
...
     --without-normal \
     --without-debug \
     --without-cxx-binding \
     --with-abi-version=5 \
+    --enable-widec \
+    --enable-pc-files \
+    --with-shared \
+    CPPFLAGS=-I$PWD/ncurses/widechar \
+    LDFLAGS=-L$PWD/lib \
     CFLAGS="-Os -s -fno-stack-protector -U_FORTIFY_SOURCE" \
```
Comment (bauen1) [inline src/minimal_overlay/bundles/ncurses/02_build.sh]: Harcoded CFLAGS, please replace this with `$CFLAGS`

**Pull request merged**